core-services/prow/02_config/_config.yaml

```diff
@@ -1468,6 +1468,24 @@ tide:
     repos:
     - openshift-priv/ocs-operator
     - openshift/ocs-operator
+  - includedBranches:
+    - ocm-2.3
+    labels:
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - downstream-change-needed
+    - bugzilla/invalid-bug
+    - backports/unvalidated-commits
+    repos:
+    - openshift/assisted-installer
+    - openshift/assisted-installer-agent
+    - openshift/assisted-service
```

Comment:
Is `openshift/assisted-test-infra` supposed to have a `ocm-2.3` branch too? The current change will result in unmergeable PRs on that repo/branch, as this query does not cover the repo, and the one on L2332 covers the repo, but excludes the branch.

Reply:
test-infra is out-of-product so it should not have this branch